src/realmGeolBasin.ttl

```diff
@@ -39,50 +40,54 @@ sorelph:hasPlanetaryStructure rdf:type owl:ObjectProperty .
 
 ###  http://sweetontology.net/realmGeolBasin/BackArcBasin
 soreagb:BackArcBasin rdf:type owl:Class ;
-                   rdfs:subClassOf soreagb:Basin ;
+                   rdfs:subClassOf soreagb:SedimentaryBasin ;
                    rdfs:label "back arc basin"@en .
 
 
-###  http://sweetontology.net/realmGeolBasin/Basin
-soreagb:Basin rdf:type owl:Class ;
```

Comment:
It looks like you are deleting classes rather than deprecating/obsoleting them?

This means that older URLs to soreagb:Basin will no longer resolve, and worse, there will be a referential integrity problem with any external data or triples that reference it, without an easy way to fix this (perhaps I am missing something and there as external set of replacement triples being maintained). 

Note that this kind of issue would be solved by opaque IDs, which allows labels to freely change so long as the definition remains the same  but when I raised this in #49 the consensus was that terminology in earth sciences was fixed and there was no drift... time to revisit?

Reply:
Hi Chris, we'll wind this one back and retain the original Basin class! Bad practice from us to break URIs.

Not that I really think there actually are **any** URIs pointing to it out there, do you know of any?